src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,164 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React, { PropsWithChildren } from 'react';
+
+import Table from './Table';
+import { defaultSortingIcons } from './Table.const';
+import { TableProps } from './Table.types';
+import { mockColumns, mockData } from './__mocks__/table';
+
+const renderTable = ({ ...rest }: PropsWithChildren<TableProps<object>>) => {
+	return render(<Table {...rest} />);
+};
+
+describe('<Table />', () => {
+	it('Should be able to render without any items', () => {
+		renderTable({
+			options: {
+				columns: mockColumns,
+				data: [],
+			},
+		});
+
+		mockColumns.forEach((col) => {
+			const text = col.Header as string;
+
+			if (text) {
+				expect(screen.getByText(new RegExp(text))).toBeDefined();
```

Comment:
Waarom zit `text` hier in een `RegExp()`?

Reply:
Partial string matching, https://jestjs.io/docs/expect#tomatchregexp--string